HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
+	if (class.name.asString.at(0) == $F) { class.name.postln }
+}; "";
```

Comment:
I was talking about adding `"";`, which IMHO should not appear in Help files (although it's nice trick).

About using `func { |foo| ... }` vs `func { arg foo; ... }` I agree with @jamshark70, I think the first one is a "cleaner" syntax (I even proposed years ago to switch to that everywhere in the Class library, in a desire to "modernize" it).